packages/block-library/src/latest-posts/index.php

```diff
@@ -171,6 +171,8 @@ function render_block_core_latest_posts( $attributes ) {
 		$class .= ' has-author';
 	}
 
+	$class .= ' begin-explicitly-added ' . gutenberg_block_supports_classes() . ' end-explicitly-added';
```

Comment:
begin-explicitly-added and end-explicitly-added are test classes right? Should be removed before the merge.

Reply:
Yeah, all changes in `block-library` were for testing purposes. :)